packages/app-accounts/src/modals/Transfer.tsx

```diff
@@ -127,6 +127,18 @@ function Transfer ({ className, onClose, recipientId: propRecipientId, senderId:
         </div>
       </Modal.Content>
       <Modal.Actions onCancel={onClose}>
+        <TxButton
+          accountId={senderId}
+          extrinsic={extrinsic}
+          icon='sign-in'
+          isBasic
+          isDisabled={!hasAvailable}
+          isSign
+          label={t('Sign Transaction')}
```

Comment:
I am actually a bit worried putting this directly only the transfer modal - the next would be to add it to all modals.

 It just seems saner to have it specifically on the actual signer. So instead of pushing through `isSign` on the specific functions (which is _everywhere_), if we put it on the signer Modal (which, is a mess) it can either go and submit or just sign instead of submitting.

Reply:
Yeah I wanted to do it like that at first but the problem is that the signer needs to show extra fields for nonce and blocks when you want to just sign so there would have to be an extra step, like:
- click 'sign' in the signer
- this puts the signer in 'sign' mode and shows extra fields
- now you click 'sign' again and this actually starts the signing process

But from what you're saying this is probably the lesser evil so I'm gonna do something like that.

Reply:
Makes sense - moved the logic inside the signer Modal.